client/src/app/pages/applications/components/application-detail-drawer/application-detail-drawer.tsx

```
@@ -267,7 +278,7 @@ const TabDetailsContent: React.FC<{
             {application?.archetypes?.length ? (
               <>
                 <DescriptionListDescription>
-                  {application.archetypes.length ?? 0 > 0 ? (
+                  {(application.archetypes.length ?? 0 > 0) ? (
```

Biome (1.9.4)

[error] Comparing to itself is potentially pointless.



(lint/suspicious/noSelfCompare)